lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -732,6 +735,26 @@ def _get_node_information(self, node: Node) -> Dict[str, str]:
             node.log.debug("detecting vm generation...")
             information[KEY_VM_GENERATION] = node.tools[VmGeneration].get_generation()
             node.log.debug(f"vm generation: {information[KEY_VM_GENERATION]}")
+            node.log.debug("detecting mana driver enabled...")
+            information[KEY_MANA_ENABLED] = node.tools[KernelConfig].is_enabled(
```

Comment:
There should be a method in nic already, please use it.

Reply:
Even inside of nic.py, it looks like this is the way mana driver is checked.
``` python
# handle situation when there is no mana driver, but have mana pci devices
if self._is_mana_device_discovered() and not self._node.tools[
    KernelConfig
].is_enabled("CONFIG_MICROSOFT_MANA"):
```

Reply:
The goal is to know whether the mana driver is enabled in distros, not to know whether the mana device is actually present. I don't think there is any code in nic.py to reuse unless I created a function just for checking the mana kernel config.

Reply:
So, it needs to be added into nic, and make names clear, including `is_mana_driver_enabled`, `is_mana_device_present`.